CSharp_Course_Project/Task1_StudentList/StudentManager.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CSharp_Course_Project.Task1_StudentList
+{
+    public class StudentManager
+    {
+        private readonly List<Student> students = new List<Student>();
```

Comment:
Please use constructor to initialize fields with initial values.

Reply:
fixed